src/simulation/test/LoadGeneratorTests.cpp

```diff
@@ -125,6 +125,36 @@ TEST_CASE("generate load with unique accounts", "[loadgen]")
             10 * Herder::EXP_LEDGER_TIMESPAN_SECONDS, false);
     }
 }
+TEST_CASE("handle soroban loadgen mode misconfiguration gracefully",
```

Comment:
Nit: This doesn't have to be it's own test, but could just be a `SECTION` in "generate soroban load"